flybywire-aircraft-a320-neo/html_ui/Pages/A32NX_Core/A32NX_Boarding.js

```diff
@@ -19,11 +19,14 @@ class A32NX_Boarding {
     }
 
     async init() {
-        // Set default pax (0)
-        await this.setPax(0);
-        await this.loadPaxPayload();
-        await this.loadCargoZero();
-        await this.loadCargoPayload();
+        const inDeveloperState = SimVar.GetSimVarValue("L:A32NX_DEVELOPER_STATE", "Bool");
+        if (!inDeveloperState) {
+            // Set default pax (0)
+            this.setPax(0);
+            this.loadPaxPayload();
+            this.loadCargoZero();
+            this.loadCargoPayload();
+        }
```

Comment:
**thought**: There used to be lots of `await` usages there. Now there are none. That _might_ be fine. But then probably you also want to remove the `async` modifier on the function?

Reply:
This case (reset payload to 0) simply sets the values, while some other usages of the same fn would ideally require an ``await`` before proceeding to cargo loading

Reply:
I'm not quite sure what you mean. As there are no `await`s, one can remove `async` from `async init() {`.

Reply:
I'd definitely keep `await` in when the function exposes a Promise (is async). You shouldn't consdier the internals of the functions from the call site, as those might change in the future.

Reply:
I think I misunderstood from the first comment about async, I get what you meant 👍🏻